docs/pages/docs/ui-components/hyperlink-toolbar.mdx

```diff
@@ -19,7 +19,7 @@ TODO Image
 
 You can change or replace the Hyperlink Toolbar with your own React component. In the demo below, a button is added to the default Hyperlink Toolbar, which opens a browser alert.
 
-<Example name="ui-components/hyperlink-toolbar-buttons" />
+[//]: # (<Example name="ui-components/hyperlink-toolbar-buttons" />)
```

Comment:
do we keep this page in docs, or remove it for now?

Reply:
It's hidden anyway so I think we may as well keep it, will be a bit less work when we add it later (after exporting default hyperlink toolbar buttons)